in.srev.guiscrcpy.yml

```diff
@@ -0,0 +1,128 @@
+---
+app-id: in.srev.guiscrcpy
+runtime: org.kde.Platform
+runtime-version: 5.15-21.08
+sdk: org.kde.Sdk
+command: guiscrcpy
+add-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    version: '21.08'
+    add-ld-path: "."
+cleanup-commands:
+  - mkdir -p ${FLATPAK_DEST}/lib/ffmpeg
+
+modules:
+  - shared-modules/libusb/libusb.json
+
+  # https://github.com/nexustar/scrcpy-flatpak
+  - name: scrcpy
+    buildsystem: simple
+    build-commands:
+      - install -D scrcpy-server-v1.20 /app/share/scrcpy-server.jar
+      - meson x --buildtype release --strip -Db_lto=true -Dprebuilt_server=/app/share/scrcpy-server.jar
+      - ninja -C x
+      - install -D x/app/scrcpy /app/bin/scrcpy
```

Comment:
if the build needs the prebuilt server, I would split that into a module and use the standard `meson` buildsystem.
if it doesn't need it at build time I'd just add it to the post-install and still use `meson` buildsystem.

in short in both case I'd use `meson` as a buildsystem.

Reply:
Yes, you are right, I have switched to the meson buildsystem.